### ISSUE TITLE
LrStats niet gedocumenteerd

### ISSUE THREAD

**Issue by HHoofs** [NetherlandsForensicInstitute/lir-deprecated#55]
Ik heb me nu al bijna een paar keer in de vingers gesneden met de LrStats.
Het zou denk ik goed zijn om:
* Te documenteren wat de verschillende metrics zijn/inhouden (en wanneer je ze wel/niet verwacht)
* De individuele LR's onder klasse 0 als inverse te rapporten (dus eigenlijk in lijn met hoe het gemiddelde voor deze klasse wordt berekend)

**Comment by geradts1** [comment]
Indeed, I would appreciate if we would like to use this in our EU project Starlight

**Comment by wowtor** [comment]
Eens dat we dit beter moeten documenteren als we hem erin willen houden.
Heeft het hiervoor voldoende meerwaarde? Anders kunnen we het ook
uitfaseren. De gebruiker zou dan de metrics zelf moeten kiezen.

Op ma 14 mrt. 2022 23:25 schreef Huub ***@***.***>:
